Aztec/Classes/Public/Extensions/NSAttributedString+Attachments.swift

```diff
@@ -22,4 +22,26 @@ extension NSAttributedString
             }
         }
     }
+
+    /// Determine the character ranges for an attachment
+    public func ranges(forAttachment attachment: NSTextAttachment) -> [NSRange]
+    {
+        // find character range for this attachment
+        let range = NSRange(location: 0, length: length)
+
+        var refreshRanges = [NSRange]()
+
+        enumerateAttribute(NSAttachmentAttributeName, inRange: range, options: []) { (value, effectiveRange, nil) in
+
+            guard let foundAttachment = value as? NSTextAttachment where foundAttachment == attachment else
+            {
+                return
+            }
+
+            // add this range to the refresh ranges
```

Comment:
The line below is actually very clear, so we can safely remove this comment too.